plugins/org.locationtech.udig.ui/src/org/locationtech/udig/ui/FeatureTypeEditor.java

```diff
@@ -807,4 +891,26 @@ public void setErrorMessage( String errorMessage ) {
         errorDecorator.showHoverText(errorMessage);
     }
 
+    /**
+     * @param editElement
```

Comment:
It would be great to see a vaild javadoc and only one `@return`  statement ;) 

However, does it make sense to check `restrictionName` for null and convert `toLowerCase()` for comparision as well?

Reply:
OK this was addressed as well as some other minor issues